ethereum-consensus/src/altair/light_client.rs

```diff
@@ -1,21 +1,69 @@
 use crate::{
     altair::{SyncAggregate, SyncCommittee},
     phase0::BeaconBlockHeader,
-    primitives::{Bytes32, Version},
+    primitives::{Bytes32, Slot, Version},
 };
 use ssz_rs::prelude::*;
 
+pub const FINALIZED_ROOT_INDEX: usize = 105;
+pub const CURRENT_SYNC_COMMITTEE_INDEX: usize = 54;
+pub const NEXT_SYNC_COMMITTEE_INDEX: usize = 55;
+
```

Comment:
```suggestion

pub const CURRENT_SYNC_COMMITTEE_INDEX_FLOOR_LOG_2: usize = 5;
```

Reply:
why not have the actual constant values? they are used [in some places](https://github.com/ethereum/consensus-specs/blob/48143056b9be031ec810912ffc3227f7443eccd9/specs/altair/light-client/sync-protocol.md#initialize_light_client_store) in the spec.